src/views/assets/Platform/PlatformList.vue

```diff
@@ -140,10 +136,7 @@ export default {
         },
         moreCreates: {
           callback: (item) => {
-            this.$router.push({
-              query: { _type: item.name, _category: item.category }
-            })
-            this.$refs.genericListTable.onCreate()
+            this.$refs.genericListTable.onCreate({ query: { type: item.name, category: item.category }})
           },
           dropdown: []
         }
```

Comment:
There are no notable code differences observed between the provided source and target versions. Both seem to follow consistent coding practices and logic flow, with no apparent errors, warnings, or optimizations to suggest at this point. However, further testing across different contexts over time may reveal any potential discrepancies or improvements in best practice usage of Vue.js components like these. Please update the context if additional information is needed.